src/scaffold.rs

```diff
@@ -209,6 +213,43 @@ pub(crate) fn verification_config() -> Result<String> {
     ))
 }
 
+pub(crate) fn artifacthub(
+    metadata_path: PathBuf,
+    version: &str,
+    questions_path: Option<PathBuf>,
+) -> Result<String> {
+    let comment_header = r#"# Kubewarden Artifacthub Package config
+#
+# Use this config to submit the policy to https://artifacthub.io.
+#
+# This config can be saved to its default location with:
+#   kwctl scaffold artifacthub > artifacthub-pkg.yml "#
+        .to_string();
+
+    let metadata_file =
+        File::open(metadata_path).map_err(|e| anyhow!("Error opening metadata file: {}", e))?;
+    let metadata: Metadata = serde_yaml::from_reader(&metadata_file)
+        .map_err(|e| anyhow!("Error unmarshalling metadata {}", e))?;
+    let questions_content: String;
+    let questions = match questions_path {
+        Some(path) => {
+            questions_content = fs::read_to_string(path)
+                .map_err(|e| anyhow!("Error reading questions file: {}", e))?;
+            Some(questions_content.as_str())
+        }
+        None => None,
+    };
```

Comment:
I also don't like this block of code for the same reasons of the `usage_contents` snippet from the `annotate.rs` file.

I think this can be solved in the same way